packages/notifi-frontend-client/lib/models/FusionEvent.ts

```diff
@@ -1,11 +1,14 @@
 import { Types } from '@notifi-network/notifi-graphql';
 
+import { ValueOrRef } from './SubscriptionCardConfig';
+
 export type FusionEventMetadata = {
   uiConfigOverride?: {
     topicDisplayName?: string;
     historyDisplayName?: string;
     icon?: Types.GenericEventIconHint;
     customIconUrl?: string;
+    subscriptionValueOrRef: ValueOrRef<string>;
```

Comment:
per our discussion, I think we also need to add 
`isSubscriptionValueSelectable?: boolean;`
here

Reply:
Thanks!!! you are right. I was misoriended. Let me change it to `isSubscriptionValueInputable?: boolean;` as it also could be be an input text field